### PR TITLE
SDIT-1700: 🐛 Attempt to reduce false positives for health endpoint

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: charts/generic-prometheus-alerts/ci/tests/ingress-alerts-tests.yaml [hunk 1]
```diff
@@ -8,9 +8,9 @@ evaluation_interval: 1m
 tests:
   - interval: 1m
     input_series:
-      - series: 'nginx_ingress_controller_requests{exported_namespace="test-application-dev", ingress="test-application-ingress", path="/bob", status="500"}'
```
Comment (petergphillips) [inline charts/generic-prometheus-alerts/ci/tests/ingress-alerts-tests.yaml:11]: these tests were wrong previously - there is only the `/` and `/health` endpoint in grafana / prometheus

File: charts/generic-prometheus-alerts/ci/tests/ingress-alerts-tests.yaml [hunk 2]
```diff
@@ -21,7 +21,7 @@ tests:
     #   which is greater than 0
     #   the average of those is 0.041
     promql_expr_test:
-      - expr: avg(rate(nginx_ingress_controller_requests{exported_namespace="test-application-dev", ingress="test-application-ingress", path!="/health", status=~"5.*"}[1m]) > 0) by (ingress, exported_namespace)
+      - expr: avg(rate(nginx_ingress_controller_requests{exported_namespace="test-application-dev", ingress="test-application-ingress", path="/", status=~"5.*"}[1m]) > 0) by (ingress, exported_namespace)
```
Comment (petergphillips) [inline charts/generic-prometheus-alerts/ci/tests/ingress-alerts-tests.yaml:24]: see above, there is only `/` or `/health` endpoints so don't need a `!=` here

File: charts/generic-prometheus-alerts/ci/tests/ingress-alerts-tests.yaml [hunk 3]
```diff
@@ -56,10 +56,10 @@ tests:
     name: 5xxErrorResponses alert not fired as path is for /health
 
     promql_expr_test:
-      - expr: avg(rate(nginx_ingress_controller_requests{exported_namespace="test-application-dev", ingress="test-application-ingress", path!="/health", status=~"5.*"}[5m]) > 0) by (ingress, exported_namespace)
+      - expr: avg(rate(nginx_ingress_controller_requests{exported_namespace="test-application-dev", ingress="test-application-ingress", path="/", status=~"5.*"}[5m]) > 0) by (ingress, exported_namespace)
         eval_time: 6m
         exp_samples:
-      - expr: avg(rate(nginx_ingress_controller_requests{exported_namespace="test-application-dev", ingress="test-application-ingress", path="/health", status=~"5.*"}[5m]) > 0) by (ingress, exported_namespace)
+      - expr: avg(rate(nginx_ingress_controller_requests{exported_namespace="test-application-dev", ingress="test-application-ingress", path="/health", status=~"5.*"}[5m]) > 0.004) by (ingress, exported_namespace)
```
Comment (petergphillips) [inline charts/generic-prometheus-alerts/ci/tests/ingress-alerts-tests.yaml:62]: tested against current false positive alerts for `hmpps-prisoner-to-nomis-update-prod`.  They all seem to be around the `0.0037` mark so `0.004` will then ignore them

Comment (petergphillips) [reply charts/generic-prometheus-alerts/ci/tests/ingress-alerts-tests.yaml:62]: also only doing the `/health` endpoint for now `/` is still at `0`

**Review by mattops — APPROVED** [review]

**Review by mattops — APPROVED** [review]

**Pull request merged**